vendor/github.com/openshift/api/operator/v1/zz_generated.swagger_doc_generated.go

```diff
@@ -205,6 +205,25 @@ func (AddPage) SwaggerDoc() map[string]string {
 	return map_AddPage
 }
 
+var map_Capability = map[string]string{
```

Comment:
See docs comments in vendor/github.com/openshift/api/operator/v1/zz_generated.crd-manifests/0000_50_console_01_consoles.crd.yaml file